### DISCUSSION TITLE
AML does not load Mods in WOTC.

### DISCUSSION THREAD

**Discussion by Talencho117** [X2CommunityCore/xcom2-launcher#347] [Help]
I have this problem with the launcher, recently I wanted to play the game again, install the launcher to divide it between the Chimera Squad and the WOTC, install LWOTC with its respective necessary mods, put the parameters in the Launcher settings and start the game, but there the problem, the launcher loads the game without any mod,   instead in Chimera Squad the launcher loads the game with at least 50 mods normally, then try to play the Xcom 2 Vanilla with mods to which only install LW2 and load perfectly, now is where I do not understand what problem the Launcher has, I deleted the parameters to see if they loaded, I put them back, review the mods reinstalling them, I downloaded the launcher again and even with everything, the game load without mods that are less than 10, so I do not know what other option to do than reinstall the game with the DLC or download an old version of the Launcher waiting if it loads them correctly. Any idea that it might be in the launcher files? :(

**Comment by RevZero** [comment]
> put the parameters in the Launcher settings 

What parameters? Make sure you dont have `-regenerateinis` enabled.

> and start the game, but there the problem, the launcher loads the game without any mod, instead in Chimera Squad the launcher loads the game with at least 50 mods normally

So it works with Chimera and it does not with XCOM?

From what you wrote, its somewhat hard to understand what exactly you are trying to communicate. The whole text is essentially one sentence 😶 You may want to spend a line break and a . from time to time.
